superset/assets/src/dashboard/components/Header.jsx

```diff
@@ -88,6 +100,36 @@ class Header extends React.PureComponent {
     this.props.setEditMode(!this.props.editMode);
   }
 
+  overwriteDashboard() {
+    const dashboardId = this.props.dashboardInfo.id;
```

Comment:
why not pull this out with the rest of the `props` destructuring you're doing below, and then just use `dashboardInfo.id` in the `onSave` call? (just strange to do partial destructuring)

Reply:
fixed.